pingora-cache/src/memory.rs

```diff
@@ -307,12 +307,10 @@ impl Storage for MemCache {
     }
 
     async fn purge(&'static self, key: &CompactCacheKey, _trace: &SpanHandle) -> Result<bool> {
```

Comment:
```suggestion
    async fn purge(&'static self, key: &CompactCacheKey, _trace: &SpanHandle) -> Result<bool> {
        // This usually purges the primary key because, without a lookup, variance key is usually
        // empty
```